save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/RunExecutionRequest.kt

```diff
@@ -23,4 +26,13 @@ data class RunExecutionRequest(
     val sdk: Sdk,
     val execCmd: String? = null,
     val batchSizeForAnalyzer: String? = null,
-)
+
+    val testingType: TestingType,
+    val contestName: String? = null,
```

Comment:
a suggestion: try to use ID instead of name to simplify searching entity on backend side

Reply:
I think that this would make API less clear, because e.g. in save-api-cli we would need to make two requests: at first get `Contest` and only then submit an execution request. Also, `contest` table will likely be not very complicated for exact method of searching to matter.